src/search/FindInFiles.js

```diff
@@ -381,6 +390,7 @@ define(function (require, exports, module) {
      * file exclusion filters, and isn't binary). Used when updating results incrementally - during the
      * initial search, these checks are done in bulk via getCandidateFiles() and the filterFileList() call
      * after it.
+     * @private
```

Comment:
For readability, its best to add a single space between the description section in a JSDoc and the declaration/parameters/return section. Not in any currently done code, just a note for the future.